src/test/fuzz/hex.cpp

```diff
@@ -27,7 +28,11 @@ FUZZ_TARGET(hex)
         assert(ToLower(random_hex_string) == hex_data);
     }
     (void)IsHexNumber(random_hex_string);
-    (void)uint256::FromHex(random_hex_string);
+    if (uint256::FromHex(random_hex_string)) {
+        assert(random_hex_string.length() == 64);
```

Comment:
I'm not sure, internally we're using `base_blob::size()` quite a bit already but I think for this test it makes sense to hardcode 64 since this function deals with user input, so we really do want to make sure this keeps expecting 64 characters. I don't think there are any other `64` constants in this PR that would benefit from being deduplicated or documented further?